c-bindings/wrapper.cpp

```diff
@@ -2,10 +2,10 @@
 #include "verifier.hpp"
 
 extern "C" {
-    bool validate_proof(const uint8_t* seed, uint8_t k, const uint8_t* challenge, const uint8_t* proof, uint16_t proof_len, uint8_t* quality_buf) {
+    bool validate_proof(const uint8_t* plot_id, uint8_t k, const uint8_t* challenge, const uint8_t* proof, uint16_t proof_len, uint8_t* quality_buf) {
         Verifier v;
-        auto quality = v.ValidateProof(seed, k, challenge, proof, proof_len);
-        if (quality.GetSize() == 0) {
+        auto quality = v.ValidateProof(plot_id, k, challenge, proof, proof_len);
+        if (quality.GetSize() != 256) {
```

Comment:
is it possible to have *fewer* than 256 bits (32 bytes) of quality? If so, you may need to print leading zeros into `quality_buf`

Reply:
According to https://github.com/Chia-Network/chiapos/blob/e5eeb6ed591cf22f67ad528d04ace2d8e635ee5f/src/verifier.hpp#L55 it should always return either exactly 32 bytes/256 bits or 0